docs/source/deploy/production_intermediate.rst

```diff
@@ -14,16 +14,15 @@ If you prefer to use PyTorch directly, feel free to use any Lightning checkpoint
 
     import torch
 
-    model = torch.load("path/to/lightning/checkpoint.ckpt")
-    model.eval()
 
-You can also pull out the specific modules you want out of the checkpoint:
```

Comment:
are you removing this because it is in the next section? (I know the code below is wrong)

Reply:
Hi @awaelchli - I am removing this not because it is working perfectly in the next section. I removing it because I think the motivation of this snippet was to show that one can load a part of the model directly from the pl checkpoint which isn't the case #13160. The first example in the doc allows the user to load the entire model using pl checkpoint. They can now load a part of the model using something like `model.encoder`.